strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/resources/artifacts/lorem-ipsum-1.txt

```diff
@@ -0,0 +1,39 @@
+Eĥo pleje liternomo hu un', si danke kilometro mallongigoj sia. Ehe it identiga parentezo. Ko pra rolfinaĵo malprofitanto, kaŭzo prepoziciaĵo veo u, jo orda neigi estiel sub. Eko gardi kunmetita da, kio jh kiel aligi nuancilo. Sis ig tempa hebrea. Us unu afrikato anstataŭi, danki aperi aspektismo et nun, it tuta tiom traigi cit. Hekto multe ci ism, tri nj mekao esperanteca.
```

Comment:
I believe it would be better to just use `new RandomInputStream(length)` and read from it, instead of having these files.